pwiz_tools/Skyline/Model/SrmDocument.cs

```diff
@@ -1509,6 +1509,17 @@ public SrmDocument ImportMassList(MassListInputs inputs,
                         Replace(TextUtil.SEPARATOR_TSV_STR, @" ");
                     errorList.Add(new TransitionImportErrorInfo(x.PlainMessage, x.ColumnIndex, x.LineNumber + 1, line));  // CONSIDER: worth the effort to pull row and column info from error message?
                 }
+                catch (Exception ex)

```

Comment:
I don't think this is the right place to put this. This is supposed to be for issues with parsing a line, not for global errors. You should be catching other exceptions higher up the callstack.

Reply:
Catching it here is advantageous as it ties into the reporting mechanism for both the UI and commandline use cases.